lib/feedback_form.dart

```diff
@@ -56,10 +58,13 @@ class _FeedbackFormState extends State<FeedbackForm> {
               });
             },
           ),
-          RaisedButton(
+          emailVerified ? RaisedButton(
             child: Text('Submit Feedback'),
             onPressed: handleSubmitFeedback,
-          )
+          ) : Text('You need to verify you account to leave feedback',
```

Comment:
There should be a button for the user to verify the email. It could lead him to another page to click verify and or something similar. I understand that it is sent automatically at sign up, but there should be an option to resend the verification email.

Reply:
U mean a button to resend email right? Okay I'll add it